extension/src/cli/RefreshXLF.ts

```diff
@@ -18,16 +18,17 @@ enum Option {
   failOnChange = "--fail-changed",
 }
 interface Parameters {
-  workspaceFolderPath: string;
+  appFolderPath: string;
+  workspaceFilePath: string | undefined;
   updateGxlf: boolean;
   failOnChange: boolean;
 }
 
 const usage = `
 Usage:
-$> node ${functionName} <path-to-al-app-folder> [${Object.values(Option).join(
-  " "
-)}]
+$> node ${functionName} <path-to-al-app-folder> [<path-to-workspace.code-workspace>] [${Object.values(
```

Comment:
Wouldn't this print
`
node RefreshXLF.js <path-to-al-app-folder> [<path-to-workspace.code-workspace>] [--update-g-xlf --fail-changed]
`

But you actually supports 
`
node RefreshXLF.js <path-to-al-app-folder> [<path-to-workspace.code-workspace>] [--update-g-xlf] [--fail-changed]
`
right?

Reply:
I remember thinking that you probably were going to point that out and then forgot about it. Good thing that you pointed it out :laughing: